mqtt.py

```diff
@@ -81,9 +81,7 @@ def stream(self, mqtt_detect_request: str = '{}'):
                             f"doods/detect/{mqtt_detect_request.id}", 
                             payload=json.dumps(detect_response.asdict(include_none=False)), qos=0, retain=False)
                     
-
-        except Exception as e:
```

Comment:
This was mainly done to get stacktraces in the logs to allow for easier debugging. It should be equivalent to catching the exception and logging since the previous code exits the thread after catching the exception